src/BuildIntegration/Microsoft.NETCore.Native.Unix.props

```diff
@@ -49,6 +49,7 @@ See the LICENSE file in the project root for more information.
     <LinkerArg Include="@(NativeLibrary)" />
     <LinkerArg Include="@(AdditionalNativeLibrary)" />
     <LinkerArg Include="-g" />
+    <LinkerArg Include="-Wl,-rpath,'$ORIGIN'" />
```

Comment:
@sergiy-k Interesting, so it seems that both syntaxes are right. We use the form with "=" in CoreCLR for setting linker options.